include/vkgcDefs.h

```diff
@@ -200,6 +200,15 @@ enum ShaderStageBit : unsigned {
   ShaderStageComputeBit = (1 << ShaderStageCompute),         ///< Compute shader bit
 };
 
+/// Enumerates LLPC types of unlinked shader elf.
+enum UnlinkedShaderTypeEnum {
+  UnlinkedShaderTypeFirst,
```

Comment:
I don't think this enum is useful unless the start value is non-zero. So currently you can always use 0 as the loop initial value.
Also, if you provide `first`, I think we always need `last`, which matches Vulkan style: UnlinkedStageFirst, UnlinkedStageLast.
And the loop could be:
`for (stage = UnlinkedStageFirst; stage <= UnlinkedStageLast; ++stage)`